Amz.py

```diff
@@ -55,10 +87,17 @@ def main():
     tree = html.fromstring(response.content)
 
     print('Aquiring list of giveway links...')
-    table = tree.xpath('//table[@id="giveaways"]/tbody/tr[@class="lucky"]/td[1]/a/@href')
+    table = tree.xpath('//table[@id="giveaways"]/tbody/tr[@class="lucky video" or @class="lucky"]/td[1]/a/@href')
+
     print("Finished! Let's get started!\n")
+
     # Sometimes the top 3 class names are not correctly named so we skip them
     fresh_links = table[2:]
+
+    if len(fresh_links) == 0:
+        time.sleep(3600)  # Sleep for one hour
```

Comment:
Nice! Though maybe a comment to explain what is going on here?

Reply:
Added.